ntoskrnl/include/internal/probe.h

```diff
@@ -1,16 +1,80 @@
+/*
+ * PROJECT:     ReactOS Kernel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Internal header containing information class probing helpers
+ * COPYRIGHT:   Copyright ???
```

Comment:
I guess you were really the first one to tentatively write properly these validation functions, so you may just put your own copyright, skipping that "unknown" entry 😂